templates/welcome.php

```diff
@@ -0,0 +1,104 @@
+<?php
+\load_template(
+	\dirname( __FILE__ ) . '/admin-header.php',
+	true,
+	array(
+		'settings' => '',
+		'welcome' => 'active',
+	)
+);
+?>
+
+<div class="privacy-settings-body hide-if-no-js">
+	<h2><?php \esc_html_e( 'Welcome', 'activitypub' ); ?></h2>
+
+	<p><?php \esc_html_e( 'ActivityPub turns your blog into a federated social network. This means you can share and talk to everyone using the ActivityPub protocol, including users of Friendica, Pleroma and Mastodon.', 'activitypub' ); ?></p>
+	<p>
+		<?php
+		\printf(
+			// translators:
+			\esc_html__(
+				'People can follow you by using the username %1$s or the URL %2$s. Users who can not access this settings page will find their username on the %3$sEdit Profile%4$s page.',
+				'activitypub'
+			),
+			\sprintf(
+				'<code>%s</code>',
+				\esc_attr( \Activitypub\get_webfinger_resource( wp_get_current_user()->ID ) )
+			),
+			\sprintf(
+				'<code>%s</code>',
+				\esc_url_raw( \get_author_posts_url( wp_get_current_user()->ID ) )
+			),
+			\sprintf(
+				'<a href="%s">',
+				\esc_url_raw( \admin_url( 'profile.php#activitypub' ) )
+			),
+			'</a>'
+		);
```

Comment:
```suggestion
		echo \wp_kses_post(
			// translators: %1$s is a username, %2$s and %3$s are URLs.
			\__(
				'People can follow you by using the username %1$s or the URL %2$s. Users who can not access this settings page will find their username on the <a href="%3$s">Edit Profile</a> page.',
				'activitypub'
			),
			\sprintf(
				'<code>%s</code>',
				\esc_html( \Activitypub\get_webfinger_resource( wp_get_current_user()->ID ) )
			),
			\sprintf(
				'<code>%s</code>',
				\esc_html( \get_author_posts_url( wp_get_current_user()->ID ) )
			),
			\esc_attr( \admin_url( 'profile.php#activitypub' ) )
		);
```
It's better to just include the HTML tags. Also added a translators comment and fixed the escaping.

Reply:
makes sense, but what about `wp_kses( ..., 'data' )` instead?

Reply:
Ah nice!